config/config-example.js

```diff
@@ -228,6 +228,11 @@ exports.repl = true;
 exports.replsocketprefix = './logs/repl/';
 exports.replsocketmode = 0o600;
 
+// allowhotpatchall - allows for hotpatching every hotpatchable part of PS
+// at once. Generally speaking, there probably won't be a need to for this,
+// which is why this is disabled by default.
+exports.allowhotpatchall = false;
```

Comment:
Should be `true` by default; Main is the only server where this should be off.

Or, well, it should be named `disablehotpatchall = false` by default.